### PR TITLE
Quickstart: Disable browser launch in containers unless override option enabled

### PR DESCRIPTION
**Pull request by psiinon**
Screenshot for when the panel is disabled:
Screenshot 2021-09-24 at 13 44 33

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties [hunk 1]
```diff
@@ -94,6 +94,9 @@ quickstart.launch.panel.default.message2 = To do this you will need to:
 quickstart.learn.panel.title			= Learn More
 quickstart.learn.panel.message1			= This screen links to local and remote resources that will help you learn more about ZAP.
 
+quickstart.panel.launch.container		= <html><p><b>You appear to be running ZAP in a container, so launching browsers has been disabled as this is unlikely to work.</b></p><p>\
+You will need to use a browser that you don't launch from ZAP, configure it to proxy through ZAP and to import the ZAP root CA certificate.</p><p>\
+If you think ZAP will be able to launch browsers in your environment then you can enable this feature via the Option: Display / Enable app integration in containers</p><p><p></html>
```
Comment (kingthorin) [inline addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties:99]:
```suggestion
If you think ZAP will be able to launch browsers in your environment then you can enable this feature via the Option: Display / Enable app integration in containers.</p><p><p></html>
```

Are the two extra starting paragraph tags on purpose?

**Review by kingthorin — COMMENTED** [review]
LGTM just needs a period.

File: addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties (hunk 1 above)
Comment (psiinon) [reply addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties]: Yes, the other original message has them. I think it helped the display when scrolling was required, but in any case I just copied the existing one ;)

**Comment by psiinon** [comment]
Have added the dot :)

File: addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties (hunk 1 above)
Comment (kingthorin) [reply addOns/quickstart/src/main/resources/org/zaproxy/zap/extension/quickstart/resources/Messages.properties]: All good, just wanted to make sure :wink:

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Pull request merged**